packages/app/src/runs/guide/TourCard.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <PromoCard
+    :title="t('runs.slideshow.tour.title')"
+    :body="t('runs.slideshow.tour.description')"
+  >
+    <template #image>
+      <RecordGuideImage />
+    </template>
+    <template #action>
+      <PromoAction
+        :action="action"
+        :left-icon="IconObjectTassel"
+        :right-label="t('runs.slideshow.controls.view')"
+      />
+    </template>
+  </PromoCard>
+</template>
+
+<script lang="ts" setup>
+import { useI18n } from '@cy/i18n'
+import PromoCard from '@packages/frontend-shared/src/gql-components/promo/PromoCard.vue'
+import PromoAction from '@packages/frontend-shared/src/gql-components/promo/PromoAction.vue'
+
+import { IconObjectTassel } from '@cypress-design/vue-icon'
+
+import RecordGuideImage from '../../assets/record-guide.svg'
+
+const { t } = useI18n()
+
+defineProps<{
+  action: () => void
```

Comment:
We've got this `defineProps` a lot. It's quite simple, but might be worth trying to find an abstraction (I suggested one, but it might not work here - I didn't try it yet).

Wouldn't  consider this to be a blocker, but food for thought - do you think these duplication is worth the effort to avoid?